### ISSUE TITLE
Adding Recurrent Neural Network algorithm to PLP package

### ISSUE THREAD

**Issue by chandryou** [OHDSI/PatientLevelPrediction#37]
Hi all,
I've tried to incorporate Recurrent Neural Network analysis for representing temporal pattern of EHR into PLP package, which I named '[CIReNN](http://www.ohdsi.org/web/wiki/lib/exe/fetch.php?media=symposium_2017:ohdsi_cirenn_poster_2017_.pdf)' 

I think I've almost done this job. But I can't figure out how I can use validation/test set from population by using index in PLP package, and which result I should make for the return value. 

Could anyone can help me for completing this work? 

You can see the [demo code](https://github.com/chandryou/PatientLevelPrediction/blob/dailydevelop/demo/CIReNN_test.r) I made.
You need to install Python, Tensorflow and Keras in your computer. And then you can install keras pacakge in R, by using backend tensorflow in python. 

Thank you!

**Comment by PRijnbeek** [comment]
Sure we can have a call next Wednesday i will email you with gotomeeting details. We can show how we did that in our deep learning code.

**Comment by jreps** [comment]
Sorry, I only just saw this post.  This code is great!  I recently added in the temporal option into the python plpData to sparse matrix conversion so should be able to add that into the R sparse matrix conversion also.  Once that is done it looks like we just need to extract the code you have written into the fit format and I think the prediction just uses the standard stats::predict?  If so, existing prediction code should be reusable.
